Cely/CelyLoginViewController.swift

```diff
@@ -32,9 +32,9 @@ class CelyLoginViewController: UIViewController {
         if let fields = textFields,
             fields.flatMap({$0}).reduce(true, {$0 && !$1.text!.isEmpty}) {
 
-            Cely.save(usernameField?.text, forKey: "username")
-            Cely.save("token 29585843899", forKey: "token", securely: true)
-            Cely.changeStatus(to: .LoggedIn)
+            if let username = usernameField?.text, let password = passwordField?.text {
+                Cely.loginCompletionBlock?(username, password)
```

Comment:
Hm, okay. Cell does know username and password at this point...

Can you just save username and password, and change login status at this point? Then user doesn't need to do them in the completion block.